modules/nodes-nodes-jobs-about.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-jobs.adoc
+
+[id='nodes-pods-configuring-replication_{context}']
+= About jobs 
+
+A job tracks the overall progress of a task and updates its status with information
+about active, succeeded, and failed pods. Deleting a job will clean up any pod
+replicas it created. Jobs are part of the Kubernetes API, which can be managed
+with `oc` commands like other object types.
+
+There are two types of jobs in {product-title}:
+
+Job::
+A regular job is a run-once object. 
+
+CronJob:: 
+
+A CronJob can be scheduled to run multiple times, use a CronJob.
+
+A _CronJob_ builds on a regular job by allowing you to specifically
+schedule how the job should be run. CronJobs are part of the
+link:http://kubernetes.io/docs/user-guide/cron-jobs[Kubernetes] API, which
+can be managed with `oc` commands like other object types.
+
+CronJobs are useful for creating periodic and recurring tasks, like running backups or sending emails. 
+CronJobs can also schedule individual tasks for a specific time, such as if you want to schedule a job for a low activity period.
+
+ifdef::openshift-online[]
+[IMPORTANT]
+====
+CronJobs are only available for _OpenShift Online Pro_. For more information about the
+differences between Starter and Pro tiers, visit the
+link:https://www.openshift.com/pricing/index.html[pricing page].
+====
+endif::[]
+
+[WARNING]
+====
+A CronJob creates a job object approximately once per execution time of its
+schedule, but there are circumstances in which it fails to create a job or
+two jobs might be created.  Therefore, jobs must be idempotent and you must
+configure history limits.
+====
+
+= About creating jobs
+
+Both types of jobs require a job configuration that consists of the following key parts:
+
+- A pod template, which describes the application the pod will create.
+- An optional `parallelism` parameter, which specifies how many pod replicas running in parallel should execute a job. If not specified, this defaults to
```

Comment:
I'd prefer to say away from replicas, they might be confusing with what's going on in Deployments. Just say 'how many pods is running in parallel at any point in time' or alike.